config/locales/static/cookies.yml

```diff
@@ -0,0 +1,22 @@
+---
+en:
+  static:
+    privacy_policy:
```

Comment:
We'd need to change this key to `cookie_policy` and create a view template so it is loaded automatically. I'll get  to this if you remind me.